R/model.R

```diff
@@ -1,7 +1,7 @@
 sgl <- function(
     bn, bs, ix, iy, nobs, nvars, x, y, pf, dfmax, pmax, nlam, flmin, ulam, eps,
     maxit, vnames, group, intr, asparse, standardize,
-    lower_bnd, upper_bnd) {
+    lower_bnd, upper_bnd, loss) {
```

Comment:
Rather than adding an argument here, can we split `sgl()` into two separate functions: 1 for ls and one for logit? That will clean up a lot of the `if...else` stuff.

Reply:
Sure!